public-ui/src/app/data-browser/views/pm/pm-react.component.tsx

```diff
@@ -0,0 +1,472 @@
+import { Component } from '@angular/core';
+import { BaseReactWrapper } from 'app/data-browser/base-react/base-react.wrapper';
+import { AgeChartReactComponent } from 'app/data-browser/charts/chart-age/chart-age-react.component';
+import {ValueReactChartComponent} from 'app/data-browser/charts/chart-measurement-values/chart-value-react.component';
+import { GENDER_STRATUM_MAP } from 'app/data-browser/charts/react-base-chart/base-chart.service';
+import { TooltipReactComponent } from 'app/data-browser/components/tooltip/tooltip-react.component';
+import { reactStyles } from 'app/utils';
+import { Spinner } from 'app/utils/spinner';
+import { environment } from 'environments/environment';
+import { Configuration, DataBrowserApi } from 'publicGenerated/fetch';
+import * as React from 'react';
+
+const styles = reactStyles({
+    pmContainer : {
+        margin: '1.2em'
+    },
+    aside: {
+        paddingRight: '18px',
+        display: 'block'
+    },
+    dbCard: {
+        width: '100vw'
+    },
+    pmLayout: {
+        display: 'flex'
+    },
+    btnLink: {
+        fontSize: '14px',
+        color: '#0077b7'
+    },
+    bsTitle: {
+        paddingTop: '1em',
+        paddingLeft: '1.5em'
+    },
+    chartLayout: {
+        display: 'flex',
+        flexWrap: 'wrap',
+        justifyContent: 'space-between',
+        alignItems: 'center',
+        paddingTop: '1em'
+    }
+});
+
+const PMGroups = [];
+PMGroups.push({group: 'blood-pressure', groupName: 'Mean Blood Pressure', concepts: [
+    {conceptId: '903118', conceptName: 'Systolic', chartType: 'histogram', analyses: []},
+    {conceptId: '903115', conceptName: 'Diastolic', chartType: 'histogram', analyses: []}
+]});
+PMGroups.push({group: 'height', groupName: 'Height', concepts: [
+    {conceptId: '903133', conceptName: 'Height', chartType: 'histogram', analyses: []}
+]});
+PMGroups.push({group: 'weight', groupName: 'Weight', concepts: [
+    {conceptId: '903121', conceptName: 'Weight', chartType: 'histogram', analyses: []}
+]});
+PMGroups.push({group: 'mean-waist', groupName: 'Mean waist circumference', concepts: [
+    {conceptId: '903135', conceptName: 'Mean waist circumference', chartType: 'histogram', analyses: []}
+]});
+PMGroups.push({group: 'mean-hip', groupName: 'Mean hip circumference', concepts: [
+    {conceptId: '903136', conceptName: 'Mean hip circumference', chartType: 'histogram', analyses: []}
+]});
+PMGroups.push({group: 'mean-heart-rate', groupName: 'Mean heart rate', concepts: [
+    {conceptId: '903126', conceptName: 'Mean heart rate', chartType: 'histogram', analyses: []}
+]});
+PMGroups.push({group: 'wheel-chair', groupName: 'Wheel chair use', concepts: [
+    {conceptId: '903111', conceptName: 'Wheel chair use', chartType: 'column', analyses: []}
+]});
+PMGroups.push({group: 'pregnancy', groupName: 'Pregnancy', concepts: [
+    {conceptId: '903120', conceptName: 'Pregnancy', chartType: 'column', analyses: []}
+]});
```

Comment:
Nit: Since we know all the objects, we can set them with the initial declaration instead of pushing them individually:
```suggestion
const PMGroups = [
  {group: 'blood-pressure', groupName: 'Mean Blood Pressure', concepts: [
    {conceptId: '903118', conceptName: 'Systolic', chartType: 'histogram', analyses: []},
    {conceptId: '903115', conceptName: 'Diastolic', chartType: 'histogram', analyses: []}
  ]},
  {group: 'height', groupName: 'Height', concepts: [
    {conceptId: '903133', conceptName: 'Height', chartType: 'histogram', analyses: []}
  ]},
  {group: 'weight', groupName: 'Weight', concepts: [
    {conceptId: '903121', conceptName: 'Weight', chartType: 'histogram', analyses: []}
  ]},
  {group: 'mean-waist', groupName: 'Mean waist circumference', concepts: [
    {conceptId: '903135', conceptName: 'Mean waist circumference', chartType: 'histogram', analyses: []}
  ]},
  {group: 'mean-hip', groupName: 'Mean hip circumference', concepts: [
    {conceptId: '903136', conceptName: 'Mean hip circumference', chartType: 'histogram', analyses: []}
  ]},
  {group: 'mean-heart-rate', groupName: 'Mean heart rate', concepts: [
    {conceptId: '903126', conceptName: 'Mean heart rate', chartType: 'histogram', analyses: []}
  ]},
  {group: 'wheel-chair', groupName: 'Wheel chair use', concepts: [
    {conceptId: '903111', conceptName: 'Wheel chair use', chartType: 'column', analyses: []}
  ]},
  {group: 'pregnancy', groupName: 'Pregnancy', concepts: [
    {conceptId: '903120', conceptName: 'Pregnancy', chartType: 'column', analyses: []}
  ]}
];
```